### PR TITLE
Limit log length

### PR DESCRIPTION
**Pull request by vimemo**
# Description

Limits pouch db message length.

medic/medic-webapp#4481

# Review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- [ ] Documented: Announced in Changes.md in plain English. Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

### CODE REVIEW

**Comment by vimemo** [comment]
hi @SCdF, please review. The pouch db json log has the following format:
{message: '', file: '', line: x}. This limits the message text to 120 characters.

File: admin/src/js/controllers/export-feedback.js [hunk 1]
```diff
@@ -9,11 +9,15 @@ angular.module('controllers').controller('ExportFeedbackCtrl',
...
     'ngInject';
 
+    var MESSAGE_LIMIT = 120;
+
     var safeStringify = function(str) {
       if (typeof str === 'string') {
         return str;
       }
       try {
+        var suffix = str.message.length > MESSAGE_LIMIT ? '...' : '';
```
Comment (SCdF) [inline admin/src/js/controllers/export-feedback.js]: Will this cover all cases? It looks like line 16 above is not going to be limited?

**Comment by vimemo** [comment]
hello @SCdF, here is an update, please review. I thought this ticket was more related to pouch logs rather than whatever the user types for feedback. But I can see now that the user might just paste a long stack trace or something else there.

**Comment by SCdF** [comment]
@vimemo sorry for the confusion around this, it's 100% OK if it was intentional, I didn't mean for my comments to sound like a forced directive to change it or anything! I was just asking because it wasn't clear to me if it was intentionally not truncated or not.

After your explanation, it sounds to me like you thought it through and your original approach is correct. Perhaps what would be worthwhile is a comment on those `if` branches making it clear (since there are no data types to help us here) that the string branch is user written feedback, and the bottom branch is automated and thus could be accidentally massive.

**Pull request merged**